lexer/lexer.go

```diff
@@ -138,12 +138,17 @@ func isLetter(ch byte) bool {
 
 // NOTE:
 // Helper function to determine if the current character is a number
-// Patams: ch (:byte)
+// Params: ch (:byte)
 // Returns: bool
 func isDigit(ch byte) bool {
 	return '0' <= ch && ch <= '9'
 }
 
+// NOTE:
+// Helper function to determine if the current character is a boolean type
+// Params: ch (:byte)
+// returns: bool
+
```

Comment:
Make the function similar to the read identifier function. Have it read and advance the position in the lexer until in encounters a byte where the value can no longer be a boolean. For example read and look for the bytes t, r, u, e or f, a, l, s, e.